packages/next/build/utils.ts

```diff
@@ -70,12 +70,13 @@ export function printTreeView(
       messages.push([
         `${symbol} ${
           item.startsWith('/_')
-            ? ' '
+            ? '  '
             : pageInfo && pageInfo.static
-            ? chalk.bold('⚡')
+            ? // '⚡' has a width of two characters in the terminal
```

Comment:
Comment probably isn't needed now since the package handles this nuanced detail.